core/src/test/resources/serialization/lioncore.json

```diff
@@ -80,48 +80,48 @@
       "concept": {
         "metamodel": "LIonCore_M3",
         "version": "1",
-        "key": "LIonCore_M3_Concept"
+        "key": "Concept"
       },
       "properties": [
         {
           "property": {
             "metamodel": "LIonCore_M3",
             "version": "1",
-            "key": "LIonCore_M3_Concept_abstract"
+            "key": "Concept_abstract"
           },
           "value": null
         },
         {
           "property": {
             "metamodel": "LIonCore_M3",
             "version": "1",
-            "key": "LIonCore_M3_NamespacedEntity_simpleName"
+            "key": "NamespacedEntity_simpleName"
```

Comment:
Hadn't we renamed this to `"name"`?

Also: these keys are still pretty qualified. Apart from the ones with obvious clashes, I'd go for the shortest, human-readable key possible. But I have to adapt the TS implementation still to see whether that's sensible.

Reply:
I renamed simpleName in name and qualified only the keys that we needed to qualify. This required a lot of fixing existing metamodels :)

Reply:
Ok, something went wrong and I lost the changes...